<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSaveData</class>
 <widget class="QDialog" name="DialogSaveData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelName">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNameOpen">
       <property name="toolTip">
        <string>Choose file name</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ssdb.qrc">
         <normaloff>:/img/file.png</normaloff>:/img/file.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelPath">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPathOpen">
       <property name="toolTip">
        <string>Choose directory path</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ssdb.qrc">
         <normaloff>:/img/open_path.png</normaloff>:/img/open_path.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="labelFormat">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>Format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxFormat">
         <property name="toolTip">
          <string>Choose file format</string>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="labelDelimiter">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>Delimiter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxDelimiter">
         <property name="toolTip">
          <string>Choose delimiter</string>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="Russian" country="Ukraine"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ssdb.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSaveData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSaveData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
